{"uid":"1c6f136385124fbc","name":"Comparing Dashboard data with List data for Vendors","fullName":"TestInventDashboard.TestInventDashboard#test_get_vendor_rpt","historyId":"2e8b555e209bd4cd53c6e61aec4b4e7f","time":{"start":1683811202361,"stop":1683811215601,"duration":13240},"status":"failed","statusMessage":"AssertionError: False != True : Verification Failed to mismatch title.","statusTrace":"self = <TestsCases.TestInventDashboard.TestInventDashboard testMethod=test_get_vendor_rpt>\n\n    @allure.title(f\"Comparing Dashboard data with List data for Vendors\")\n    @allure.severity(allure.severity_level.NORMAL)\n    def test_get_vendor_rpt(self):\n        self.LoginMethod.nativelogin(self.driver, \"frugal@latido.com.np\", \"Test@123\")\n        try:\n            self.assertEqual(self.InventDash.get_full_vendor_rpt(), True, msg=\"Verification Failed to mismatch title.\")\n        # Checking if assertion failed\n        except AssertionError as e:\n            allure.attach(self.driver.get_screenshot_as_png(), name=\"Failed_for_customer\", attachment_type=AttachmentType.PNG)\n            self.log.error(f\"Assertion failed here while finding element. {str(e)}\")\n>           raise e\n\nTestInventDashboard.py:63: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nTestInventDashboard.py:58: in test_get_vendor_rpt\n    self.assertEqual(self.InventDash.get_full_vendor_rpt(), True, msg=\"Verification Failed to mismatch title.\")\nC:\\Program Files\\JetBrains\\PyCharm Community Edition 2022.3.3\\plugins\\python-ce\\helpers\\pycharm\\teamcity\\diff_tools.py:33: in _patched_equals\n    old(self, first, second, msg)\nE   AssertionError: False != True : Verification Failed to mismatch title.","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_unittest_setUpClass_fixture_TestInventDashboard","time":{"start":1683811171351,"stop":1683811171351,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"hasContent":false,"shouldDisplayMessage":false,"attachmentsCount":0}],"testStage":{"status":"failed","statusMessage":"AssertionError: False != True : Verification Failed to mismatch title.","statusTrace":"self = <TestsCases.TestInventDashboard.TestInventDashboard testMethod=test_get_vendor_rpt>\n\n    @allure.title(f\"Comparing Dashboard data with List data for Vendors\")\n    @allure.severity(allure.severity_level.NORMAL)\n    def test_get_vendor_rpt(self):\n        self.LoginMethod.nativelogin(self.driver, \"frugal@latido.com.np\", \"Test@123\")\n        try:\n            self.assertEqual(self.InventDash.get_full_vendor_rpt(), True, msg=\"Verification Failed to mismatch title.\")\n        # Checking if assertion failed\n        except AssertionError as e:\n            allure.attach(self.driver.get_screenshot_as_png(), name=\"Failed_for_customer\", attachment_type=AttachmentType.PNG)\n            self.log.error(f\"Assertion failed here while finding element. {str(e)}\")\n>           raise e\n\nTestInventDashboard.py:63: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nTestInventDashboard.py:58: in test_get_vendor_rpt\n    self.assertEqual(self.InventDash.get_full_vendor_rpt(), True, msg=\"Verification Failed to mismatch title.\")\nC:\\Program Files\\JetBrains\\PyCharm Community Edition 2022.3.3\\plugins\\python-ce\\helpers\\pycharm\\teamcity\\diff_tools.py:33: in _patched_equals\n    old(self, first, second, msg)\nE   AssertionError: False != True : Verification Failed to mismatch title.","steps":[],"attachments":[{"uid":"b1cf93e5d9a8100a","name":"Failed_for_customer","source":"b1cf93e5d9a8100a.png","type":"image/png","size":141591},{"uid":"69c7ec999be1c26d","name":"log","source":"69c7ec999be1c26d.txt","type":"text/plain","size":154}],"parameters":[],"stepsCount":0,"hasContent":true,"shouldDisplayMessage":true,"attachmentsCount":2},"afterStages":[{"name":"_unittest_setUpClass_fixture_TestInventDashboard::0","time":{"start":1683811215617,"stop":1683811215617,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"hasContent":false,"shouldDisplayMessage":false,"attachmentsCount":0}],"labels":[{"name":"feature","value":"Inventory Dashboard"},{"name":"severity","value":"normal"},{"name":"suite","value":"TestInventDashboard"},{"name":"subSuite","value":"TestInventDashboard"},{"name":"host","value":"0day"},{"name":"thread","value":"9148-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"TestInventDashboard"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"1c6f136385124fbc.json","parameterValues":[]}